src/Support/Compatibility/MiddlewareBridge.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support\Compatibility;
+
+interface MiddlewareBridge
```

Comment:
Not sure about the name `MiddlewareBridge`.
Looking at what it does, it is more of a repository, I think I'll prefer the name `MiddlewareRepository`.
@spawnia Any input on this?

Reply:
Is the term _Repository_ not usually used for a pattern of database access, e.g. as an alternative to the _Active Record_ approach? In any case, I'm rather new to both PHP and Laravel, so I'm not yet acquainted with the Lingo :)

Reply:
How about `MiddlewareAdapter`?

Reply:
Repositories are used to store/retrieve data, while hiding away where the data is actually coming from. For example Laravel uses repositories for Cache and Config and I think they also do for authentication. 
However I think that `MiddlewareAdapter` is just as good.

Reply:
I'm replacing all occurrences of "bridge" with "adapter", if you guys can think of an even more fitting term, feel free to adapt the code later.